.azure-pipelines/apiview-sync-staging.yml

```diff
@@ -0,0 +1,31 @@
+parameters:
```

Comment:
We should try to eventually move all these pipeline files under eng\pipelines.

cc @chidozieononiwu

Reply:
I will submit a new PR to make this change.